CHANGELOG.md

```diff
@@ -1,14 +0,0 @@
-# Changelog
-All notable changes to this project will be documented in this file.
-
-## [Unreleased]
-
-## [0.0.1] - 2020-08-14
-- Added pre-footer links to previous visualizations
-- Updated SVG Years
-- Basic Context Slider Functionality
-- Created Components out of the sections in the viz
-- Changed favicon to USGS
-- expanded data definition for title attributes in Visualization.vue and HeaderUSWDSSelection.vue
```

Comment:
## [1.1.0] - 2024-07-16
- Vue dependency updated to v3, with behind the scenes changes to adopt composition API